ci/docker/debian-experimental-cpp.dockerfile

```diff
@@ -73,6 +73,7 @@ RUN if [ -n "${gcc}" ]; then \
         libutf8proc-dev \
         libxml2-dev \
         libxsimd-dev \
+        libxxhash-dev \
```

Comment:
Without this I had linker errors with `xxhash`. Though I'm surprised this _isn't_ needed for debian 12?

Reply:
Ah, sid/experimental's `liblz4.so` uses `libxxhash.so`.